yarn 2.lock

```diff
@@ -0,0 +1,12469 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn 2.lock? What is up with this?

Reply:
Might have been added when I've reinstalled yarn (possibly?!). Shall I remove it?

Reply:
Yes please. We already have a lock file.